recipes/fltk/all/conanfile.py

```diff
@@ -68,10 +70,13 @@ def requirements(self):
         self.requires("libjpeg/9e")
         self.requires("libpng/1.6.40")
         if self.settings.os in ["Linux", "FreeBSD"]:
-            self.requires("opengl/system")
-            self.requires("glu/system")
+            if self.options.with_gl:
+                self.requires("opengl/system")
```

Comment:
Since this is only for Linux and FreeBSD, is it possible to require `libglvnd` instead of `opengl/system`?

Reply:
@jwillikers I'm currently not using opengl, so I don't know if I can confirm the correct operation. Maybe keep that for a next iteration?

Reply:
Yeah, that's alright. Thanks!